src/sicmutils/env/sci.cljc

```diff
@@ -1,66 +1,114 @@
+   'sicmutils.abstract.function                (ns-publics 'sicmutils.abstract.function)
+   'sicmutils.calculus.basis                   (ns-publics 'sicmutils.calculus.basis)
+   'sicmutils.calculus.coordinate              (ns-publics 'sicmutils.calculus.coordinate)
+   'sicmutils.calculus.covariant               (ns-publics 'sicmutils.calculus.covariant)
+   'sicmutils.calculus.derivative              (ns-publics 'sicmutils.calculus.derivative)
+   'sicmutils.calculus.form-field              (ns-publics 'sicmutils.calculus.form-field)
+   'sicmutils.calculus.manifold                (ns-publics 'sicmutils.calculus.manifold)
+   'sicmutils.calculus.map                     (ns-publics 'sicmutils.calculus.map)
+   'sicmutils.calculus.vector-field            (ns-publics 'sicmutils.calculus.vector-field)
+   'sicmutils.expression.compile               (ns-publics 'sicmutils.expression.compile)
+   'sicmutils.expression.render                (ns-publics 'sicmutils.expression.render)
+   'sicmutils.mechanics.lagrange               (ns-publics 'sicmutils.mechanics.lagrange)
+   'sicmutils.mechanics.hamilton               (ns-publics 'sicmutils.mechanics.hamilton)
+   'sicmutils.mechanics.rigid                  (ns-publics 'sicmutils.mechanics.rigid)
+   'sicmutils.mechanics.rotation               (ns-publics 'sicmutils.mechanics.rotation)
+   'sicmutils.numerical.derivative             (ns-publics 'sicmutils.numerical.derivative)
+   'sicmutils.numerical.quadrature             (ns-publics 'sicmutils.numerical.quadrature)
+   'sicmutils.numerical.ode                    (ns-publics 'sicmutils.numerical.ode)
+   'sicmutils.numerical.minimize               (ns-publics 'sicmutils.numerical.minimize)
+   'sicmutils.numerical.interpolate.polynomial (ns-publics 'sicmutils.numerical.interpolate.polynomial)
+   'sicmutils.numerical.interpolate.rational   (ns-publics 'sicmutils.numerical.interpolate.rational)
+   'sicmutils.numerical.interpolate.richardson (ns-publics 'sicmutils.numerical.interpolate.richardson)
+   'sicmutils.numerical.multimin.nelder-mead   (ns-publics 'sicmutils.numerical.multimin.nelder-mead)
+   'sicmutils.numerical.unimin.bracket         (ns-publics 'sicmutils.numerical.unimin.bracket)
+   'sicmutils.numerical.unimin.brent           (ns-publics 'sicmutils.numerical.unimin.brent)
+   'sicmutils.numerical.unimin.golden          (ns-publics 'sicmutils.numerical.unimin.golden)})
 
-  (eval '(->TeX (simplify (+ (square (sin (square 'x)))
-                             (square (cos 'x))))))
+(def ^{:doc "Default sci context options required (currently only `:namespace`
+  bindings) required to evaluate SICMUtils forms from inside of an SCI
+  context. Pass these to `sci/init` to generate an sci context."}
+  context-opts
+  (let [ns-map (into {}
+                     (map (fn [[k v]] [k (sci-ns v)]))
+                     ns-map)
+        with-macros (merge-with merge ns-map macros/ns-bindings)]
```

Comment:
For nextjournal (and other downstream consumers) it might be good to also expose the plain sci namespace map (what `with-macros` is here). Without any assumptions of what the `user` ns is. Maybe this is what the `ns-map` var could be turned into?

Reply:
yes, good call, that's much better. I made that fix and moved the `'user` change into the nextjournal test PR.